components/style/values/specified/position.rs

```diff
@@ -486,3 +490,179 @@ impl From<GridAutoFlow> for u8 {
         result
     }
 }
+
+#[cfg_attr(feature = "gecko", derive(MallocSizeOf))]
+#[derive(Clone, Debug, PartialEq)]
+/// Single template area
+pub struct TemplateAreas {
+    /// `named area` containing for each single template area
```

Comment:
I'm not sure what "named area containing" is supposed to mean... @nox, you implemented this, any nice idea about how to document this?

Reply:
Hmm... after reading the spec, I think this attribute means each `NamedArea` parsed from the `string`s.
Maybe this comment is not good for documentation.
Please teach me how to make it better! 😄